neo4j/README.md

```diff
@@ -1,67 +1,54 @@
-# Neo4j Integration
+# Agent Check: neo4j
 
 ## Overview
 
-Get metrics from Neo4j service in real time to:
-
-- Visualize and monitor Neo4j states.
-- Be notified about Neo4j failovers and events.
+This check monitors [neo4j][1] through the Datadog Agent.
 
 ## Setup
 
-The Neo4j check is **NOT** included in the [Datadog Agent][1] package.
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
 
 ### Installation
 
-If you are using Agent v6.8+ follow the instructions below to install the Neo4j check on your host. See the dedicated Agent guide for [installing community integrations][2] to install checks with the [Agent prior v6.8][3] or the [Docker Agent][4]:
-
-1. [Download and launch the Datadog Agent][1].
-2. Run the following command to install the integrations wheel with the Agent:
+To install the neo4j check on your host:
 
-   ```shell
-   datadog-agent integration install -t datadog-neo4j==<INTEGRATION_VERSION>
-   ```
-
-3. Configure your integration like [any other packaged integration][5].
+1. Install the [developer toolkit](https://docs.datadoghq.com/developers/integrations/new_check_howto/#developer-toolkit) on any machine.
```

Comment:
You should leave this current section, Datadog users will be able to install your integration automatically with `datadog-agent integration install -t datadog-neo4j==4.0.0` once this PR is merged